src/generator/synth.ts

```diff
@@ -60,6 +60,11 @@ export async function synth(options: SynthOptions = {}) {
   if (!token) {
     throw new Error('please include a GITHUB_TOKEN');
   }
+  // only set these while running in the GitHub Actions environment
+  if (process.env.GITHUB_ACTIONS) {
+    await execa('git', ['config', 'user.email', 'yoshi-automation@google.com']);
```

Comment:
you might consider doing this as a step in the action, this is what I did here:

https://github.com/conventional-changelog/conventional-changelog/blob/master/.github/workflows/ci.yaml#L15

Not a blocker for me, but seems potentially action specific.